README.md

```diff
@@ -149,6 +149,18 @@ DJANGO_REST_LOOKUP_FIELD = 'custom_email_field'
 ```
 into Django settings.py file.
 
+## Configurable Response
+
+To configurable response messages need to add following properties into settings.py
```

Comment:
Please change to `For optional configurable response messages, add the following properties to settings.py`